code/modules/atmospherics/gasmixtures/reactions.dm

```diff
@@ -236,7 +238,7 @@
 
 			energy_released += FIRE_PLASMA_ENERGY_RELEASED * (plasma_burn_rate)
 
-			cached_results["fire"] += (plasma_burn_rate) * (1 + oxygen_burn_rate)
+			cached_results[/datum/gas_reaction/plasmafire] = (plasma_burn_rate) * (1 + oxygen_burn_rate)
```

Comment:
I see why you wanted to make subtype reactions now.
This is somewhat annoying.

Reply:
Would like profiles if only so I can look at them and be sad, it's unlikely to really matter one way or the other because the hottest path in here, fire, already has it

Reply:
Dropped two cans of plasma to a hot room. This is what i got
![image](https://user-images.githubusercontent.com/54709710/143677621-00c403db-3621-48be-97b5-14ea844cf14d.png)
"master" is misleading, I meant without adding to reaction_results at all.

Reply:
thanks, I appreciate it